Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Models/Memberships.cs

```diff
@@ -15,6 +15,11 @@ public Memberships()
         public string MembershipType { get; set; }
         public DateTime MembershipDuration { get; set; }
```

Comment:
Why is this variable called `Duration` when it is a `DateTime`?  A `DateTime` indicates a point in time from which a duration can be calculated?

Reply:
this is the intention, allowing the admin to create memberships and set a duration the membership runs for

Reply:
I think what @DreyerChris is saying is that a duration is usually an int or float or something. But with DateTime it's a specific set time (you'll have to compare it with a previous date first to get the duration). Otherwise maybe you should change the variable name, or at least add a comment to explain it.

Reply:
Yeah exactly @grootendorste, I would suggest either making the duration a `int` for number of days or something or calling the field `MembershipExpiry` or something similar

Reply:
okay int does make more sense